src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -685,4 +694,18 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    private String generateCoOwnersEmailsLine(String courseId) {
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
+        StringBuffer coOwnersEmailsList = new StringBuffer();
```

Comment:
Use StringBuilder since it is more efficient and concurrency access is not required.